src/puppetlabs/trapperkeeper/services/authorization/authorization_core.clj

```diff
@@ -79,6 +86,31 @@
       (add-acl m)
       (add-query-params m)))
 
+(defn in-valid-methods?
+  "Is the given method contained in the list of valid methods?"
+  [m]
+  (let [method (str/lower-case m)]
```

Comment:
Hmmm, do we ever normalize the method from the config file to lowercase?  Seems like that would happen in the `method` private function above it were happening, but not sure that it is.